packages/flutter/lib/src/animation/animations.dart

```diff
@@ -439,12 +439,14 @@ class CurvedAnimation extends Animation<double> with AnimationWithParentMixin<do
         final double transformedValue = activeCurve.transform(t);
         final double roundedTransformedValue = transformedValue.round().toDouble();
         if (roundedTransformedValue != t) {
-          throw FlutterError(
-            'Invalid curve endpoint at $t.\n'
-            'Curves must map 0.0 to near zero and 1.0 to near one but '
-            '${activeCurve.runtimeType} mapped $t to $transformedValue, which '
-            'is near $roundedTransformedValue.'
-          );
+          throw FlutterError.fromParts(<DiagnosticsNode>[
+            ErrorSummary('Invalid curve endpoint at $t.'),
+            ErrorDescription(
+              'Curves must map 0.0 to near zero and 1.0 to near one but '
+              '${activeCurve.runtimeType} mapped $t to $transformedValue, which '
+              'is near $roundedTransformedValue.'
+            )
+          ]);
```

Comment:
I thought FlutterError would automatically take its first line and use it as a summary and use the rest as a description, is that not the case?

Reply:
Yes I believe so too, this is actually redundant change
cc: @jacob314

Reply:
Yes go ahead and revert this change as part of your followup cleanup CL.